services/src/main/java/org/apache/druid/cli/CliRouter.java

```diff
@@ -80,44 +79,39 @@ public CliRouter()
         new QueryRunnerFactoryModule(),
         new JettyHttpClientModule("druid.router.http", Router.class),
         JettyHttpClientModule.global(),
-        new Module()
-        {
-          @Override
-          public void configure(Binder binder)
-          {
-            binder.bindConstant().annotatedWith(Names.named("serviceName")).to("druid/router");
-            binder.bindConstant().annotatedWith(Names.named("servicePort")).to(8888);
-            binder.bindConstant().annotatedWith(Names.named("tlsServicePort")).to(9088);
+        binder -> {
```

Comment:
I'm not sure an anonymous class conversion to a lambda makes code better in this case. You lose visibility into the fact that this lambda is a `Module` and that the method is `configure()`, making the code more cryptic, IMO.